docs/ARCHITECTURE.md

```diff
@@ -34,9 +34,9 @@ The dynamic gas fee calculation is handled by the following pieces of code:
 
 1. [opcodes.json](../opcodes.json) defines the user input metadata in `dynamicFee` section of the opcodes, where each field is represented by a `number` or `boolean` with their corresponding labels. The app then builds the gas calculation UI form dynamically. The same happens for the precompiled contracts with [precompiled.json](../precompiled.json).
 
-1. The actual gas calculation is performed by the [util/gas.ts#calculateDynamicFee](../util/gas.ts) based on the selected [EthereumJS Common](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/common) object, via the get `param` method.
+2. The actual gas calculation is performed by the [util/gas.ts#calculateDynamicFee](../util/gas.ts) based on the selected [EthereumJS Common](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/common) object, via the get `param` method.
```

Comment:
This should not be changed, markdown is setting the numbers itself.

Reply:
@peroket haha, It's so strange. But I think it's more precise after changed, and [it](https://github.com/xiaolou86/evm.codes/blob/typo/docs/ARCHITECTURE.md) is also correctly rendered.

Reply:
Of course it's up to you ^_^

Reply:
It's easier if we don't have to track the numbers manually :-)

Reply:
@peroket yeah, it's more easier and AI. I have reverted it.